lib/config-initializer.js

```diff
@@ -44,6 +44,9 @@ function processAnswers(answers) {
     config.rules.quotes = [2, answers.quotes];
     config.rules["linebreak-style"] = [2, answers.linebreak];
     config.rules.semi = [2, answers.semi ? "always" : "never"];
+    if (answers.extendDefault) {
```

Comment:
Hmm, that's definitely a good point. I'm convinced default-on is fine. Anyone else have strong feelings either way on this?